metrics/ci_integrations/lib/source/buildkite/adapter/buildkite_source_client_adapter.dart

```diff
@@ -138,8 +138,8 @@ class BuildkiteSourceClientAdapter implements SourceClient {
       startedAt: build.startedAt ?? build.finishedAt ?? DateTime.now(),
       buildStatus: _mapStateToBuildStatus(build.state),
       duration: _calculateJobDuration(build),
-      workflowName: pipelineSlug,
-      url: build.webUrl ?? "",
+      workflowName: pipelineSlug ?? '',
```

Comment:
```suggestion
      workflowName: pipelineSlug,
```
It seems like `pipelineSlug` comes from configurations that check it to be non-null. Moreover, the workflow name is not a required field for the build according to the Firestore rules.